### ISSUE TITLE
CDU Legs направление поворотов по схеме

### ISSUE THREAD

**Issue by soitanen** [ponemetski/extrinofms-issues#127]
В базе данных есть направление поворотов по схемам выхода/захода (бывает, что поворачивают не по кратчайшему расстоянию). Там такой тег - <Sp_Turn>Left</Sp_Turn>. Учитывается ли он? Ну и учитывается ли в процедурах тег <Flytype>Fly-over</Flytype>?

**Comment by ponemetski** [comment]
не учитывается

**Comment by ponemetski** [comment]
> крен на разворот считается как разница курсов умноженная на 0.5, но не более 23 градусов
это в LNAV?

**Comment by soitanen** [comment]
> > крен на разворот считается как разница курсов умноженная на 0.5, но не более 23 градусов
> 
> @soitanen, это в LNAV?

Да, чтобы при маленьких доворотах все комфортно было, да и вообще - крен 30 не комфортно для пассажира.
